src/algo/thermal-loop/l500-thermal-loop.cpp

```diff
@@ -0,0 +1,80 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
+//// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "l500-thermal-loop.h"
+
+namespace librealsense {
+namespace algo {
+namespace thermal_loop {
+namespace l500 {
+
+
+using calib_info = l500_thermal_loop::rgb_thermal_calib_info;
+
+calib_info l500_thermal_loop::parse_thermal_table( const std::vector< byte > & data )
+{
+    rgb_thermal_calib_info res;
+    std::vector< float > float_vac;
```

Comment:
`vac` -> `vec`?
But why not simply reinterpret the data as `float const *`? Why copy it to another vector?

Reply:
Done